### PR TITLE
Update kv quantization and inference guide

### PR DESCRIPTION
**Pull request by lvhan028**
after #1377

### CODE REVIEW

File: docs/en/quantization/kv_quant.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+- Volta architecture (sm70): V100
+- Turing architecture (sm75): 20 series, T4
+- Ampere architecture (sm80, sm86): 30 series, A10, A16, A30, A100
+- Ada Lovelace architecture (sm89): 40 series
+
+In the next section, we will take `internlm2-chat-7b` model as an example, introducing the usage of kv quantization and inference of lmdeploy. But before that, please install lmdeploy from source according to the [build](../build.md) guide, because lmdeploy hasn't released this feature yet.
+
+## Usage
+
+Applying KV quantization and inference via LMDeploy is quit straightforward; simply set the `quant_policy` parameter.
```
Comment (zhyncs) [inline docs/en/quantization/kv_quant.md:20]: typo `;`

**Comment by zhulinJulia24** [comment]
@zhulinJulia24 react testcase

**Comment by zhulinJulia24** [comment]
https://github.com/InternLM/lmdeploy/blob/main/docs/zh_cn/supported_models/supported_models.md  这个文档要更新么？

**Comment by lvhan028** [comment]
> https://github.com/InternLM/lmdeploy/blob/main/docs/zh_cn/supported_models/supported_models.md 这个文档要更新么？

要的。多谢多谢，我等下更新

File: docs/en/quantization/kv_quant.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+- Turing architecture (sm75): 20 series, T4
+- Ampere architecture (sm80, sm86): 30 series, A10, A16, A30, A100
+- Ada Lovelace architecture (sm89): 40 series
+- Hopper architecture (sm90): H100, H200
+
+In the next section, we will take `internlm2-chat-7b` model as an example, introducing the usage of kv quantization and inference of lmdeploy. But before that, please install lmdeploy from source according to the [build](../build.md) guide, because lmdeploy hasn't released this feature yet.
+
+## Usage
+
+Applying kv quantization and inference via LMDeploy is quit straightforward. Simply set the `quant_policy` parameter.
```
Comment (lzhangzz) [inline docs/en/quantization/kv_quant.md:21]:
```suggestion
Applying kv quantization and inference via LMDeploy is quite straightforward. Simply set the `quant_policy` parameter.
```

File: notebook.md
Comment (RunningLeon) [inline notebook.md:1]: this file is empty

Comment (lvhan028) [reply notebook.md:1]: removed. It is checked in by mistake

**Comment by zhulinJulia24** [comment]
https://github.com/InternLM/lmdeploy/blob/3e9166d17f1917bdc7799ff4dfe99905e3543f96/docs/zh_cn/quantization/w4a16.md

这个中文的文档也可改internlm -> internlm2

File: docs/en/supported_models/supported_models.md [hunk 3]
```diff
@@ -2,21 +2,26 @@
...
-|        QWen        | 1.8B - 72B |    Yes    |   Yes   |  Yes  |
-|      QWen1.5       | 1.8B - 72B |    Yes    |   Yes   |  Yes  |
-|      QWen-VL       |     7B     |    Yes    |   Yes   |  Yes  |
-|      Baichuan      |     7B     |    Yes    |   Yes   |  Yes  |
-|     Baichuan2      |     7B     |    Yes    |   Yes   |  Yes  |
-|     Code Llama     |  7B - 34B  |    Yes    |   No    |  No   |
-|         YI         |  6B - 34B  |    Yes    |   No    |  No   |
-|   InternVL-Chat    |     -      |    Yes    |   No    |  No   |
+| Model              | Size       | FP16/BF16 | KV INT8   | KV INT4   | W4A16 |
+| ------------------ | ---------- | --------- | --------- | --------- | ----- |
```
Comment (zhulinJulia24) [inline docs/en/supported_models/supported_models.md:6]: image

**Review by lzhangzz — APPROVED** [review]

**Pull request merged**